.github/workflows/release_notes.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh -ue
+#
+# This script generates the release notes "func-e" for a specific release tag.
+# .github/release_notes.sh v0.3.0
+
+tag=$1
+prior_tag=$(git tag -l 'v*'|sed "/${tag}/,+10d"|tail -1)
+range="${prior_tag}..${tag}"
+
+git config log.mailmap true
+changelog=$(git log --format='%h %s %aN, %(trailers:key=co-authored-by)' "${range}")
+contributors=$(git shortlog -s --group=author --group=trailer:co-authored-by "${range}" | cut -f 2)
```

Comment:
maybe better sort this alphabetically? 😄

Reply:
sgtm

Reply:
I mean that way I'm always on top 😎